lms/db/__init__.py

```diff
@@ -123,6 +123,17 @@ def make_engine(settings):
     return sqlalchemy.create_engine(settings["sqlalchemy.url"])
 
 
+class CustomSession(Session):
+    @cached_property
```

Comment:
3.8! yeah

Reply:
We have our next DO right here. Remove all instances of `@lru_cache + @property`